githubdatapipeline/issues/destination.py

```diff
@@ -1,12 +1,13 @@
-from utils.db import SupabaseInterface
+from utils.db import SupabaseInterface, PostgresORM
 import sys
 def hasCommunityLabel(labels):
     if any([label["name"].lower() == "c4gt community" for label in labels]):
         return True
     return False
 
-def recordIssue(issue):
-    currentTickets = SupabaseInterface.get_instance().readAll(table="community_program_tickets")
+async def recordIssue(issue):
+    # currentTickets = SupabaseInterface.get_instance().readAll(table="community_program_tickets")
+    currentTickets = await PostgresORM().readAll(table="community_program_tickets")
```

Comment:
Where are we saving the creds for postgres? I do not see the samples.env getting updated?